src/storage/src/keyspace.rs

```diff
@@ -166,6 +166,30 @@ impl<S: StateStore> Keyspace<S> {
         Ok(strip_prefix_iterator)
     }
 
+    pub async fn prefix_iter_with_range<R, B>(
+        &self,
+        prefix_key: Vec<u8>,
```

Comment:
Really suggest naming this like `prefix_hint`. The current name sounds confusing to me.

Reply:
How about just adding an argument to `iter_with_range`?